Add loading attribute for audio and video (#43434)
mdn/content: files/en-us/web/api/htmlmediaelement/index.md

  - : Returns a boolean that indicates whether the media element has finished playing.
- {{domxref("HTMLMediaElement.error")}} {{ReadOnlyInline}}
  - : Returns a {{domxref("MediaError")}} object for the most recent error, or `null` if there has not been an error.
- {{domxref("HTMLMediaElement.loading")}}

>   - : A string providing a hint to the browser used to optimize loading the document by determining whether to load the media immediately (`eager`) or on an as-needed basis (`lazy`). See [`<video loading>`](/en-US/docs/Web/HTML/Reference/Elements/video#loading) and [`<audio loading>`](/en-US/docs/Web/HTML/Reference/Elements/audio#loading) HTML attributes for more information.

Reviewer: FWIW I tend to use hint in the detail description - unless it is important at the this level that readers know that the browser could choose to ignore the setting.

```suggestion
  - : A string indicating whether the browser should load the media immediately (`eager`) or when it is needed (`lazy`).
     See [`<video loading>`](/en-US/docs/Web/HTML/Reference/Elements/video#loading) and [`<audio loading>`](/en-US/docs/Web/HTML/Reference/Elements/audio#loading) HTML attributes for more information.
```
